### PR TITLE
Doc: give all EPSG potential names for lon_0/lat_0/x_0/y_0 (fixes #3904)

### LINKED ISSUE #3904: tmerc.html: conflicting +lon_0 definitions
On https://proj.org/en/9.3/operations/projections/tmerc.html#cmdoption-arg-lon_0 we see,
> `+lon_0=<value> Longitude of projection center.`

then later below,

> `Option +lon_0 sets the central meridian.`

Maybe both should say

> +lon_0 sets the projection center, through which the central meridian passes.

### PR DESCRIPTION
**Pull request by rouault**

### CODE REVIEW

File: docs/source/operations/options/lon_0.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .. option:: +lon_0=<value>
 
-    Longitude of projection center.
+    Longitude of central meridian/longitude of natural origin, longitude of
```
Comment (rouault) [inline docs/source/operations/options/lon_0.rst]:
```suggestion
    Central meridian/longitude of natural origin, longitude of
```

**Review by busstoptaktik — COMMENTED** [review]
I'm not a native English speaker, but I believe "semantic" should be "semantics" everywhere it appears in this PR

File: docs/source/operations/options/lat_0.rst [hunk 2]
```diff
@@ -1,6 +1,8 @@
 .. option:: +lat_0=<value>
 
-    Latitude of projection center.
+    Latitude of natural origin, latitude of false origin or latitude of
+    projection centre (naming and semantic depend on the
```
Comment (rouault) [inline docs/source/operations/options/lat_0.rst]:
```suggestion
    projection centre (naming and meaning depend on the
```

File: docs/source/operations/options/lon_0.rst [hunk 3]
```diff
@@ -1,6 +1,8 @@
 .. option:: +lon_0=<value>
 
-    Longitude of projection center.
+    Central meridian/longitude of natural origin, longitude of
+    origin or longitude of false origin (naming and semantic depend on the
```
Comment (rouault) [inline docs/source/operations/options/lon_0.rst]:
```suggestion
    origin or longitude of false origin (naming and meaning depend on the
```

File: docs/source/operations/options/x_0.rst [hunk 4]
```diff
@@ -1,5 +1,6 @@
 .. option:: +x_0=<value>
 
-    False easting, in meters.
+    False easting, easting at false origin or easting at projection centre (naming
+    and semantic depend on the projection method). Always in meters.
```
Comment (rouault) [inline docs/source/operations/options/x_0.rst]:
```suggestion
    and meaning depend on the projection method). Always in meters.
```

File: docs/source/operations/options/y_0.rst [hunk 5]
```diff
@@ -1,5 +1,6 @@
 .. option:: +y_0=<value>
 
-    False northing, in meters.
+    False northing, northing at false origin or northing at projection centre (naming
+    and semantic depend on the projection method). Always in meters.
```
Comment (rouault) [inline docs/source/operations/options/y_0.rst]:
```suggestion
    and meaning depend on the projection method). Always in meters.
```

**Comment by rouault** [comment]
> I'm not a native English speaker, but I believe "semantic" should be "semantics" everywhere it appears in this PR

switching to "meaning".  Semantic is an adjective, and semantics is the "study of meanings in a language"

**Pull request merged**